pom.xml

```diff
@@ -26,6 +28,62 @@
         <module>test-plugin</module>
     </modules>
 
+	<dependencyManagement>
```

Comment:
Hey one question @rogerwang1102  why is this?

Reply:
I use Dependency management to manage dependencies for a single project. 
Since there are 3 inherited projects under this one, I choose to centralize the management of dependency versions without adding dependencies which are inherited. (Mainly to exclude gson for inherited projects)
But since only the tests use gson, so I also added the gson in inherited pom.

Reply:
No sure about this 🤔  why is not the runtime team updating this dependecy ?

Reply:
Actually they don't need to. 
This gson is from 5 different runtime jars, not only one jar.
Their side is using correct latest version.
For example, in mule-API, we use 1.1.1 mule-api (not latest) and it uses gson 2.8.1.
But from mule-metadata, we also use older version, so the gson here is not latest. 

So we got gson from 5 runtime jars:
mule-module-service
mule-extensions-api-persistence
mule-metadata-model-json
mule-api
mule-module-extensions-spring-support

So I exclude the gson from these 5 jars and then have our own gson version.

Reply:
Got it